src/mackerel-container-agent-definition.ts

```diff
@@ -11,15 +10,21 @@ import {
   ServiceRole,
 } from "./types"
 
-export interface Props
-  extends Pick<
-    ContainerDefinitionProps,
-    Exclude<keyof ContainerDefinitionProps, "image">
-  > {
+export interface Props extends Omit<ContainerDefinitionProps, "image"> {
   apiKey: string
   roles?: ReadonlyArray<ServiceRole>
   ignoreContainer?: string
   hostStatusOnStart?: MackerelHostStatus
+  image?: ContainerImage
+}
+
+export const MackerelContainerAgentImage = {
+  Latest: ContainerImage.fromRegistry(
+    "mackerel/mackerel-container-agent:latest"
+  ),
+  Plugins: ContainerImage.fromRegistry(
```

Comment:
I think `plugins` lacks some contexts in constrast of `latest` that includes no additional plugins, however, this naming is derived from mackerel-container-agent so this matter is not of this library...

Reply:
Hmm.

For example `golang:windowsservercore` is actually "Windows version of `golang:latest`" and I think it's acceptable, therefore the naming `plugin` is not so bad...
ref: https://hub.docker.com/_/golang